elasticreindexer/process/buffer.go

```diff
@@ -0,0 +1,66 @@
+package process
+
+import "bytes"
```

Comment:
you can reference this component from the **elastic-indexer-go** repository

Reply:
will do this in a future PR